apps/pipeline/mix.exs

```diff
@@ -45,7 +45,7 @@ defmodule Pipeline.MixProject do
       {:dialyxir, "~> 1.0.0-rc.6", only: :dev, runtime: false},
       {:placebo, "~> 2.0.0-rc2", only: [:dev, :test, :integration]},
       {:smart_city_test, "~> 2.2.1", only: [:test, :integration]},
-      {:divo, "~> 1.1", only: [:dev, :integration]},
```

Comment:
same as alchemist, missing mix increment of app version number. since divo was updated in pipeline, pipeline needs a new mix version